plugins/lookup/rhub_api.py

```diff
@@ -0,0 +1,79 @@
+__metaclass__ = type
+
+
+DOCUMENTATION = """
+module: rhub_api
+short_description: Interacts with Resource Hub API.
+description:
+    - Retrieve data from Resource Hub API.
+options:
+    _terms:
+      description:
+        - API endpoint path.
+      required: true
+      type: str
+    addr:
+      description:
+        - Resource Hub base adress.
```

Comment:
Spelling of 'adress' should be 'address'.  Do a grep -R for this as this spelling is in other places in the code.